openfl-docker/README.md

```diff
@@ -0,0 +1,89 @@
+# Using OpenFL within a Container
+
+OpenFL can be used within a container for simulating Federated Learning experiments, or to deploy real-world experiments within Trusted Execution Environments (TEEs).
+
+## Base Image
+
+To develop or simulate experiments within a container, build the base image (or pull one from docker hub).
+
+```shell
+# Pull latest stable base image
+$> docker pull intel/openfl
```

Comment:
Is this image automatically uploaded on each OpenFL release?

Reply:
I think the scripts [here](https://github.com/securefederatedai/openfl/tree/develop/scripts) are used to publish latest wheels and images.

Following 1.7, OpenFL base image will come bundled with Gramine.

Reply:
> Just in case, have you tested the graminization workflow with this image, rather than the one built locally?

I'm not sure of the publishing workflow either. We should revisit this.

Dockerize with Gramine won't work with any older commits/tags/releases for graminized workspaces. Part of the process is copying manifest/makefiles from the source code (which means it must be at least the latest commit that this PR points to).

I agree that this is a little painful as we have to wait for next major release until users can benefit from this. Best would be if users always git clone the tip of develop.